samples/cpp/benchmark_app/infer_request_wrap.hpp

```diff
@@ -190,7 +190,7 @@ class InferRequestsQueue final {
                 try {
                     std::rethrow_exception(inferenceException);
                 } catch (const std::exception& ex) {
-                    throw ex;
+                    throw;
                 }
```

Comment:
What's the trick behind this? Ones of std::exception kind are rethrown, other ones just go. @tomdol take a look here plizz.

Reply:
Agree, why do we need to catch something?

Reply:
Please remove try-catch, otherwise exception message is missing

Reply:
Removed!

Reply:
@PiotrKrzem Have you checked whether it is working? I mean simulate exception in CPU/GPU and check the output

Reply:
Yes, the second ticket linked is precisely about the issue you mentioned :) The error message is correctly printed in the console.